cypress/integration/inventory/search/browse-contributors-exact.spec.js

```diff
@@ -0,0 +1,69 @@
+import uuid from 'uuid';
+import getRandomPostfix from '../../../support/utils/stringTools';
+
+import permissions from '../../../support/dictionary/permissions';
+import inventorySearch from '../../../support/fragments/inventory/inventorySearch';
+import browseContributors from '../../../support/fragments/inventory/search/browseContributors';
+import topMenu from '../../../support/fragments/topMenu';
+import users from '../../../support/fragments/users/users';
+
+describe('Search: browse contributors with exact match query', () => {
+  const testData = {};
+  const instance = {
+    source: 'FOLIO',
+    title: `Test_title_A_${getRandomPostfix()}`,
+    contributors: [
+      {
+        name: `Test_contributor_A_${getRandomPostfix()}`,
+        contributorNameTypeId: '2b94c631-fca9-4892-a730-03ee529ffe2a',
```

Comment:
It's hardcode id.Can we use uiid()?

Reply:
yes) sorry it is actually in draft status